src/main/resources/doNothingRegistrator-example.xml

```diff
@@ -1,31 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hazelcast id="hazelcast-consul-discovery" 
-           xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.6.xsd"
+<hazelcast id="hazelcast-consul-discovery"
+           xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-4.0.xsd"
            xmlns="http://www.hazelcast.com/schema/config"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <group>
```

Comment:
why is this removed, its used in the example/travis i thought no?

Reply:
The <group> tag has been removed in 4.0 and replaced by <cluster-name>

Reply:
The group tag has been removed in 4.0 and replaced by cluster-name